hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1347,6 +1362,10 @@ message ListKeysResponse {
     repeated KeyInfo keyInfo = 2;
 }
 
+message ListKeysLightResponse {
+    repeated BasicKeyInfo basicKeyInfo = 2;
```

Comment:
We should add a field indicating whether the response was truncated due to the key limit, like `isTruncated` from S3 [ListObjectsV2](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html). Really I think this needs to be added to all listing APIs in OM and SCM but we can start here.

Reply:
Thank you @errose28 for the suggestion. I have incorporated the changes, could you please take a look?